FCCee/IDEA/compact/IDEA_o1_v02/IDEA_o1_v02.xml

```diff
@@ -52,7 +52,7 @@
 
   <!-- Import Endcap plate absorber -->
   <include ref="EndPlateAbsorber_o1_v01.xml"/>
-
+  
```

Comment:
Not a major comment but it's a good practice to revert your changes using `git checkout -- <filename>` or equivalent instead of removing texts by hands... (so that you don't screw up commit histories)